Formula/manticoresearch.rb

```diff
@@ -18,16 +18,18 @@ class Manticoresearch < Formula
 
   depends_on "boost" => :build
   depends_on "cmake" => :build
-  depends_on "icu4c" => :build
```

Comment:
Why are we removing this?

Reply:
Because Manticore Search builds its own version of icu library and links with it statically. It can work w/o global ICU. There were some problems with some version of ICU, that's why we decided to build it ourselves.

Reply:
I'm afraid this goes against Homebrew policy, so we don't want to do this: https://docs.brew.sh/Acceptable-Formulae#stuff-that-requires-vendored-versions-of-homebrew-formulae

Reply:
OK, got it, thank you. Let me think what we can do.